.github/workflows/checks.yml

```diff
@@ -3,10 +3,12 @@ on: [pull_request]
 jobs:
   tests:
     runs-on: ubuntu-latest
+    outputs:
+      all: ${{ steps.changes.outputs.all }}
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4]
+        php: [7.3, 7.4]
```

Comment:
@lauraquellmalz ILIAS 6 still supports PHP 7.2, so I am not sure if we should remove this here.

Reply:
Reverted.